Jenkinsfile

```diff
@@ -246,6 +246,59 @@ for(String envKeyName: context.env.keySet() as String[]){
         }
     }
 
+    // Backup (w/ pg_dump) before deploying
+    //if (("DEV".equalsIgnoreCase(stageDeployName) || isCD) {
+    if (isCI || isCD) {
+        _stage("Pre-deploy db backup - ${stageDeployName}", context){
+            node('master') {
+
+                String PROJECT = 'moe-gwells-test'
+                String POD_DB = sh(
+                    script:
+                        """
+                            oc get pods -n "${PROJECT}" | grep -i "Running" | grep -Eo \
```

Comment:
Take a look into the [OpenShift](https://jenkins-moe-gwells-tools.pathfinder.gov.bc.ca/job/gwells/pipeline-syntax/globals#openshift) global variable provided as part of [OpenShift Jenkins Client Plugin](https://github.com/openshift/jenkins-client-plugin). Remember that you can use labels to get a much smaller subset of pods, e.g.: `oc get pods -l deploymentconfig=gwells-pgsql-dev-pr-690`, you can get the exact suffix (e.g.: `-dev-pr-690` by using `context.deployments[envKeyName].dcSuffix`